### ISSUE TITLE
Quick Start Guide Editing

### ISSUE THREAD

**Issue by yonahbg** [gwu-libraries/sfm-ui#693]
- Make sure that critical information is available on the site itself.
-  Asking and answering questions with answers in plain English (non-techie speak whenever possible) and whenever it would be valuable provide annotated screen shot.
- A user should know what each section contains when they look at the TOC on the left.
- If headers were collapsed, make sure a user would know what each section contains
- Start with
o What is SFM really?
o How can SFM be used?
o What value can SFM have (why is it amazing?)
- Information should only pertain to the first time: What are the prerequisites and what is your first collection/harvest like (beware that they don’t know these terms so you’d even want to define these)
- Don’t say “Hopefully you’ve considered…” instead help them through an initial set of considerations.
- Definition of collections that exists is circular. A collection is a group of “harvests” (e.g., your social media search criteria across platforms) centered around a theme or research topic that pull in data over a period of time.
- Beware of “2016 U.S. Elections” as it becomes dated. 
- Consider annotating images (arrows and supplemental text) and providing captions
- Whenever possible, add alternative text for users who use screen readers.
- Avoid “techie speak” like “the part of the API from which the social media is to be collected” – this can make people tune out.
- Instead of “the collection types supported by SFM include”  You can learn more about the following collection types that are supported by SFM (in this way, you are subtly explaining the action).
- For bullets like those under “Each collection harvest type has specific options, which may include” – consider starting each bullet with a bolded keyword: Schedule:, Media type: etc.
- Perhaps show the words on the left and the image on the right. 
- Add seeds 
o Define seeds in plain English at the top of the section – “Seeds are the search queries that you set up…” or something to that effect. 
o Provide caption for image and show it after basic information is provided.
o Remove table as it’s not a “quick start”
- Start harvesting
o First define harvest in plain English.
o When you’re to start, turn on your collection.
o Once you turn on…
o You can find out when the next harvest is scheduled.
o Find results for your harvest by… (what do you have to do)
- During harvesting
o Get rid of first two paragraphs and relabel section as “How do I monitor harvesting?”
o Break out a separate section on “How do I edit an existing harvest?” and then stress “you must turn off…”
- Exploring… section
o Just talk about basic exports
o Pull out all the advanced user stuff and put in a separate section of the documentation.
   (Provide a single-line link to this information at the bottom of the quick start)
o Clean up the Excel screenshot of the export, and provide a caption

**Comment by yonahbg** [comment]
@lwrubel @justinlittman I'm starting to think that this should be reconstituted as a general User Guide rather than just a quick start guide. It would include a "Getting Started" section, but also be an easy resource for user reference, like a Help guide. 

What are your thoughts?

**Comment by yonahbg** [comment]
I also think it would be useful to wait to see what we change with the UX, since that will change how things are done, to an extent.

**Comment by kerchner** [comment]
Resolved in #751
